sdcm/gemini_thread.py

```diff
@@ -86,42 +82,44 @@ def _generate_gemini_command(self):
         test_nodes = ",".join(self.test_cluster.get_node_cql_ips())
         oracle_nodes = ",".join(self.oracle_cluster.get_node_cql_ips()) if self.oracle_cluster else None
 
-        cmd = "/$HOME/{} --test-cluster={} --outfile {} --seed {} ".format(self.gemini_cmd.strip(),
-                                                                           test_nodes,
-                                                                           self.gemini_result_file,
-                                                                           seed)
+        cmd = "{} --test-cluster={} --outfile {} --seed {} ".format(self.gemini_cmd.strip(),
+                                                                    test_nodes,
+                                                                    self.gemini_result_file,
+                                                                    seed)
         if oracle_nodes:
             cmd += "--oracle-cluster={} ".format(oracle_nodes)
         if table_options:
             cmd += " ".join([f"--table-options \"{table_opt}\"" for table_opt in table_options])
         self.gemini_commands.append(cmd)
         return cmd
 
-    def run(self):
-        # pylint: disable=consider-using-with
-        executor = concurrent.futures.ThreadPoolExecutor(max_workers=len(self.loaders.nodes))
-        for loader_idx, loader in enumerate(self.loaders.nodes):
-            self.futures.append(executor.submit(self._run_gemini, loader, loader_idx))
+    def _run_stress(self, loader, loader_idx, cpu_idx):
+
+        cpu_options = ""
+        if self.stress_num > 1:
+            cpu_options = f'--cpuset-cpus="{cpu_idx}"'
+        gemini_version = self.params.get('gemini_version') or '1.7.6'
+        gemini_version = '1.7.6' if gemini_version == 'latest' else gemini_version
```

Comment:
May be set '1.7.6'  as a property ?

Reply:
so it won't be repeated ?

the whole idea was to still support `gemini_version==latest`, when I think of it, we should remove it from all the configurations. and make `1.7.6` the default

Reply:
we can configure gemini version in default.yaml which will point to latest build docker image.
once we build new one, we just need only update default.yaml with correct version.
Yes, with docker image we don't upload latest gemini release from site, so there is no point to have 'latest'.